UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs

```diff
@@ -80,6 +80,8 @@ private static TileLocation GetPooledTile()
 		public BetterBounds? GlobalCachedBounds;
 
 		[NonSerialized] public Matrix4x4 localToWorldMatrix = Matrix4x4.identity;
+		[NonSerialized] public Matrix4x4 worldToLocalMatrix = Matrix4x4.identity;
```

Comment:
useful in future, Since its thread safe and theoretically faster than the current method, tho I did try using it but it hadn't been set by The time stuff started using it